### ISSUE TITLE
TS error "Property 'label' does not exist" on Item component due to @react-stately's ItemProps typing

### ISSUE THREAD

**Issue by tinaboyce** [pingidentity/astro#1]
I've started noticing it within the last 24 hours (CI creates a new package-lock.json file so it could be a dependencies having an update that caused this to happen)

Replicate

1. Remove package-lock.json and */node_modules directory
2. `npm install`
3. Use the following code
```
import { Item } from '@pingux/astro'
<Item
    key={item.key}
    textValue={item.label}
    label={item.label}
>
    {item.children}
</Item>
```
4. Run "tsc" or "tsc --noEmit" OR use VS Code typescript extension

to get error
```
Type '{ children: Element[]; key: string; textValue: string; label: string; }' is not assignable to type 'IntrinsicAttributes & ItemProps<unknown>'.
  Property 'label' does not exist on type 'IntrinsicAttributes & ItemProps<unknown>'.ts(2322)
```

![Screenshot 2023-09-07 at 9 17 41 am](https://github.com/pingidentity/astro/assets/12929562/81c7c514-7dee-405a-9acc-bc21b2d530e5)

**Comment by e-cooper** [comment]
Thank you, we will investigate and update you when we've released a fix.

**Comment by e-cooper** [comment]
This fix was released in 2.18.0
